### PR TITLE
fix(editor): Record sessionStarted telemetry event in Setting Store

### PR DESCRIPTION
**Pull request by fedme**
## Summary
Restores the "Session started" telemetry event that was accidentally removed in https://github.com/n8n-io/n8n/pull/10945/commits/d62cfca170fbd583437cf63984ef6c6c2ea14a59#diff-056a9fd49f06d2d7a23d984175528aa82fddc8cb23e91a2cc4b8d35c3c674d6cL249

## Related Linear tickets, Github issues, and Community forum posts

[ADO-2744](https://github.com/n8n-io/n8n/blob/05467fd101da8cfe71585a145765bba5f17b9097/packages/cli/src/abstract-server.ts#L242)

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [x] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [x] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/stores/settings.store.test.ts [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+	})),
+}));
+
+vi.mock('@/stores/versions.store', () => ({
+	useVersionsStore: vi.fn(() => ({
+		setVersionNotificationSettings: vi.fn(),
+	})),
+}));
+
+const mockSettings: FrontendSettings = {
```
Comment (fedme) [inline packages/editor-ui/src/stores/settings.store.test.ts:56]: Is there a better way to mock this huge object?

Comment (netroy) [reply packages/editor-ui/src/stores/settings.store.test.ts]: pushed a commit to address this.

**Review by netroy — APPROVED** [review]

File: packages/editor-ui/src/stores/settings.store.test.ts (hunk 1 above)
Comment (fedme) [reply packages/editor-ui/src/stores/settings.store.test.ts]: Thank you!

**Review by mutdmour — APPROVED** [review]
LGTM thanks for adding tests as well

**Comment by janober** [comment]
Got released with [`n8n@1.64.2`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.64.2)

**Pull request merged**